server/routes/contacts/manage/identities/delete/manageContactDeleteIdentityController.ts

```diff
@@ -29,6 +28,19 @@ export default class ManageContactDeleteIdentityController implements PageHandle
       )
     }
 
+    res.render('pages/contacts/manage/confirmDeleteIdentity', { identityNumber })
+  }
+
+  POST = async (
```

Comment:
Just wondering can this be named as DELTE? so it matches the delete operation?

Reply:
well it's a form POST so it matches that instead. All of the controllers just have `GET` and `POST` as part of the interface.